addOns/selenium/src/main/javahelp/org/zaproxy/zap/extension/selenium/resources/help/contents/options.html

```diff
@@ -61,6 +61,17 @@ <h3>Binaries</h3>
 	<p>
 	More details about the bundled WebDrivers can be found on the overview page.
 
+	<h3>Browser Extensions</h3>
+
+	You can add, remove, enable and disable browser extensions via the relevant buttons on this screen.
+	Supported browsers and file extensions are:
+	<ul>
+	<li>Firefox: .xpi
+	<li>Chrome: crx
```

Comment:
Should the Chrome one have a period? (I guess that might apply elsewhere int his PR too.)